model/patch/patch.go

```diff
@@ -188,10 +188,15 @@ type PatchSet struct {
 }
 
 type TriggerInfo struct {
-	Aliases              []string    `bson:"aliases,omitempty"`
-	ParentPatch          string      `bson:"parent_patch,omitempty"`
-	ChildPatches         []string    `bson:"child_patches,omitempty"`
-	DownstreamParameters []Parameter `bson:"downstream_parameters,omitempty"`
+	Aliases              []string     `bson:"aliases,omitempty"`
+	ParentPatch          string       `bson:"parent_patch,omitempty"`
+	ChildPatches         []ChildPatch `bson:"child_patches,omitempty"`
```

Comment:
I think changing the type of this field will break the CLI when it tries to pull down the patch (as we discussed)

I think if we want to add this we'd need to 1) create a new field with a different name, 2) file a ticket to remove the old field and schedule that ticket for a while from now after we're confident everyone's updated their CLI.